src/main/java/io/reactivex/Single.java

```diff
@@ -1294,7 +1296,7 @@
      * </dl>
      * @param <T> the common value type
      * @param <R> the result value type
-     * @param sources the Iterable sequence of SingleSource instances
+     * @param sources the Iterable sequence of SingleSource instances, must not be empty
```

Comment:
"An empty sequence will result in an `onError` signal of `NoSuchElementException`."